zeroex/asset_data_decoder.go

```diff
@@ -20,6 +20,16 @@ const ERC721AssetDataID = "02571792"
 // ERC1155AssetDataID is the assetDataId for ERC721 tokens
 const ERC1155AssetDataID = "a7cb5fb7"
 
+// StaticCallAssetDataID is the assetDataId for staticcalls
```

Comment:
Non-blocking: I used `assetDataId` here to keep it consistent with the rest of the file; however, we refer to these as `assetProxyIds` in the monorepo. I'd be in favor of following that convention instead.